test/test_quantized.py

```diff
@@ -1865,6 +1894,30 @@ def test_qnnpack_tanh(self, X):
             self.assertEqual(qYserver, qY_hat,
                              message="QNNPACK TanH failed (FBGEMM ref)!")
 
+    """Tests the correctness of the quantized::qnnpack_sigmoid op."""
+    @given(X=hu.tensor(shapes=hu.array_shapes(1, 5, 1, 5),
+                       qparams=hu.qparams(dtypes=torch.quint8)))
+    def test_qnnpack_sigmoid(self, X):
+        # Note: In QNNPACK the output scale and zero_point can only be
```

Comment:
Suggest adding a test that sweeps the input over the range [-4,4] with a scale of 8/256 and a zero-point of 128 to ensure that we check that the LUT implementation in qnnpack and the FBGEMM implementation are identical in numerics. Alternately,  we could also implement FBGEMM qnnpack with a LUT based approach.